translation/en/RedirectedResources/assets/abdata/h/list/00_00/personality_voice_c00_03_00/translation.txt

```diff
@@ -313,11 +313,11 @@
 //んちゅっ！　ちゅぱ！　これ、最高……でしょ？　ちゅぱっ！　私も……同じ気持ちになれてるわ！=
 //ちゅっ！　んちゅっ！　上も下も、繋がってるわ！　んちゅっ、れろれろっ=
 //ちゅっ、ちゅぅぅっ、キスだけでもイッちゃいそうよぉ……ちゅぱっ、じゅるるっ=
-//だんだん気持ちよくなってきたわ……私もイケそうよ=
+//だんだん気持ちよくなってきたわ……私もイケそうよ=It's slowly starting to feel good... I'll cum soon
 //これがセックスの快感……わかってきたわ=
 //身体が早くイキたがってるわ……♪　そろそろ……ねっ？=
 //そのままおちんちんで子宮にキスして……トントンしてぇ=
-//もうすぐセックスでイケるのね……嬉しいわ=
+//もうすぐセックスでイケるのね……嬉しいわ=We are doing sex... I'm so happy
```

Comment:
I'll change it to "We're going to have sex soon... I'm so happy." then.

Reply:
Ah, I misread it. More literally she's saying "Going to cum from sex soon..."
Although セックス sticks out to the listener since it's an English word, it might be better to get rid of it in the translation because it sounds awkward including it. 
I'd go with "Gonna cum soon, huh... I'm so happy."